ksp_plugin/plugin.cpp

```diff
@@ -1307,69 +1255,6 @@ void Plugin::ReadCelestialsFromMessages(
   }
 }
 
-not_null<std::unique_ptr<DiscreteTrajectory<Navigation>>>
-Plugin::RenderBarycentricTrajectoryInNavigation(
-    DiscreteTrajectory<Barycentric>::Iterator const& begin,
-    DiscreteTrajectory<Barycentric>::Iterator const& end) const {
-  auto trajectory = make_not_null_unique<DiscreteTrajectory<Navigation>>();
-
-  NavigationFrame const& plotting_frame = *GetPlottingFrame();
-
-  if (target_ && !begin.trajectory()->Empty() &&
-      (target_->vessel->prediction().Empty() ||
-       begin.trajectory()->last().time() >
-           target_->vessel->prediction().last().time())) {
-    // NOTE(egg): this is an ugly hack to try to get a long enough trajectory
-    // while retaining a timeout.
-    auto parameters = target_->vessel->prediction_adaptive_step_parameters();
-    parameters.set_max_steps(begin.trajectory()->Size());
-    target_->vessel->set_prediction_adaptive_step_parameters(parameters);
-    target_->vessel->UpdatePrediction(current_time_ + prediction_length_);
```

Comment:
This `UpdatePrediction` disappeared; it was an ugly hack, but removing it is probably a bug.